tests/test_buildoptions.py

```diff
@@ -0,0 +1,118 @@
+import pytest
```

Comment:
consider adding a test for filtered and valid build options in the same parse call